app/controllers/aircrafts_controller.rb

```diff
@@ -25,11 +25,11 @@ def create
 
     respond_to do |format|
       if @aircraft.save
-        format.html { redirect_to aircraft_url(@aircraft), notice: "Aircraft was successfully created." }
-        format.json { render :show, status: :created, location: @aircraft }
+        format.html { redirect_to(aircraft_url(@aircraft), notice: I18n.t("notices.aircraft.created")) }
```

Comment:
Interesting.. we haven't supported internationalization yet. We will likely look to support this when we address it too. Cool!